services/app-web/src/testHelpers/apolloMocks/contractGQLMock.ts

```diff
@@ -417,7 +419,7 @@ const submitContractMockError = ({
 
 const indexContractsMockSuccess = (
     submissions: Contract[] = [
-        { ...mockContractPackageUnlockedWithUnlockedType(), id: 'test-id-123', __typename: 'Contract' },
+        { ...mockContractPackageUnlockedWithUnlockedType(), id: 'test-id-123', __typename: 'Contract', reviewStatus: 'UNDER_REVIEW' },
```

Comment:
**suggestion**: We could add `reviewStatus: 'UNDER_REVIEW'` to the default data in `mockContractPackageUnlockedWithUnlockedType`

Reply:
We can't because `unlockedContractType` doesn't have a `reviewStatus` field. In this example submissions is type `Contract[]` which does have `reviewStatus`

Reply:
Since unlockedContractType now has a reviewStatus, I went ahead and consolidated the change the mock

Reply:
Oh, I see what you're saying. That makes sense, though this brings up the issue of our queries return types.

**Issue**: I think the root issue here isn't that the `unlockedContractType` doesn't contain `reviewStatus`, but some of our queries should return `Contract` or `UnlockedContract`. In this `indexContractsMockSuccess` we are extending the unlocked contract with the  `reviewStatus`, which doesn't pass the smell test because `IndexContractsPayload` doesn't return that type, if it did we wouldn't need to add `reviewStatus`. 

**Suggestion:** We should update `IndexContractsPayload` to return either `Contract` or `Unlocked Contract`.

We don't have to address it in this PR, but we should take care of this in one of the other API tickets in this epic. Let me know what you think.

cc: @macrael

Reply:
Yeah that makes sense to me. I don't think the ticket belongs in this approvals epic though. Seems more like something for the contract API epic to me